packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -147,17 +151,12 @@ export const OdieSendMessageButton = () => {
 	const onKeyUp = useCallback( () => {
 		// Only triggered when the message is empty
 		// used to remove validation message.
```

Comment:
I think that this comment can be removed, since `onKeyUp` will be triggered on the keyup textarea event, not only when the message is empty.